servicetalk-encoding-api/src/main/java/io/servicetalk/encoding/api/IdentityContentCodec.java

```diff
@@ -17,16 +17,15 @@
 
 import io.servicetalk.buffer.api.Buffer;
 import io.servicetalk.buffer.api.BufferAllocator;
+import io.servicetalk.buffer.api.CharSequences;
 import io.servicetalk.concurrent.api.Publisher;
 
-import static io.servicetalk.buffer.api.CharSequences.newAsciiString;
-
 /**
  * Default, always supported NOOP 'identity' {@link ContentCodec}.
  */
 final class IdentityContentCodec implements ContentCodec {
 
-    private static final CharSequence NAME = newAsciiString("identity");
+    private static final CharSequence NAME = CharSequences.newAsciiString("identity");
```

Comment:
nit: we prefer to keep this as static import since the name is self-explanatory.